projects/js-packages/react-data-sync-client/src/DataSync.ts

```diff
@@ -345,7 +345,7 @@ export class DataSync< Schema extends z.ZodSchema, Value extends z.infer< Schema
 			url = this.maybeRequestDisabled( url );
 			const result = await fetch( url, args );
 			if ( ! result.ok ) {
-				throw new DataSyncError( result.statusText, {
+				throw new DataSyncError( `DS request failed: ${ result.status } - ${ result.statusText }`, {
```

Comment:
If there is no response body, this looks a bit weird.

<img width="698" alt="image" src="https://github.com/user-attachments/assets/a3f6577d-67fd-4dda-94d1-5c4d3d873010" />

```suggestion
				let errorText = result.status;
				if( result.statusText ) {
					errorText = `${errorText} - ${result.statusText}`;
				}
				throw new DataSyncError( `DS request failed: ${errorText}`, {
```